app/components/edit-claim-conditions/EditClaimConditions.tsx

```
@@ -0,0 +1,238 @@
+
+      console.log({ receipt });
+      toast.success('Updating Claims status', {
+        description: `Updating successful with ${receipt.status}`,
+        duration: 3000,
+      });
+    } catch (err) {
+      setIsProcessing(false);
+      setIsError(true);
+
+      toast.error('Updating Claim Conditions', {
+        description: (err as Error).message || `Updating failed!`,
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)